8.x/theme-development/helpers.md

```diff
@@ -1,100 +1,144 @@
 ## Template helper functions
 
 [[toc]]
-With every template engine, the content of the website can be managed. In order for the template creators to be able to work with the content more easily, for example: to receive the content, modify it, display the body in different pages of the website, functions called **Helper** are created. These functions are also created in the Larammerce Template Engine.
 
-Let's check the provided helpers one by one:
+According to the need of every single content management engine, to simplify the process of pulling, modifying, and displaying changes for template creators, some special functions called **Helpers** are defined. 
+The Larammerce engine is no exception.
+Therefore, for a better understanding, this document is provided for you, explaining the use of each helper function.
+To reach all the helpers, direct to the path: `app/Utils/CMS/Template/helpers.php`.
 
-First, open the helper file: `app/Utils/CMS/Template/helpers.php`.
+**NOTE:** Check for updates in the helpers file, due to any changes in functions.
 
-This file has 1462 code lines. This section describes all the helpers. Some of these helpers are old, some new, and some will be removed in the next version, which are specified as @deprecated in the phpdoc above them.
 
+---
 
+### How to setup a play ground
 
-### `unparse_url`
-```php
-unparse_url(array $parsed_url, array $ommit = []): string
-```
-This function unparses the URL. As you know, there is a `pars_url` function in PHP, which works as follows:
+To test these functions, add a new blade file to the `larammerce-base-them` directory, then follow these steps:
+
+
+1. Create a directory in the project root from the admin panel.
+2. Click the plus button located at the bottom left of the page.
+3. Enter a title and URL, and enable the "has webpage" option in the form.
+- Note: the URl name should be same as the blade file name.
+4. Save the changes.
+5. After saving the changes, a new form will display. Choose "editing webpage" from the dropdown menu and select the created blade file name.
+6. Save and exit.
+7. Write your desired functions (they are mentioned as `Example` in this document) in the blade file, run `./deploy.sh` in ypur terminal in the `larammerce-base-them` directory and view their results on the specified URL.
+
+![create webpage in admin panel](/helpers/CreatePage.png)
+
+---
+
+### Helper functions description
+
+#### parse_url 
+
+*This is a php function which parses a URL and returns an associative array containing any of the various components of the URL that are present(such as `scheme`, `user`. `pass`, `host`, `port`,`path`, `query`, `fragment`). The values of the array elements are not URL decoded.* *<sup>[1](#1)</sup>*
```

Comment:
This is a php function -> This is a built-in php function
`user`. `pass` -> `user`, `pass`